requirements.txt

```diff
@@ -0,0 +1,4 @@
+# python >= 3.6
```

Comment:
use requirements.txt and virtualenv to make life easier for your team-mates (and for later you)
You can also specify python version in shebang
#!/bin/env python3.6 
(in fact you should to if files intended to be run from command line)